.eslintrc.cjs

```diff
@@ -3,9 +3,4 @@ module.exports = {
   // This tells ESLint to load the config from the package `eslint-config-agreeto`
   // https://turbo.build/repo/docs/getting-started/create-new#understanding-eslint-config-custom
   extends: ["agreeto"],
```

Comment:
This tripped me up for a second because I thought how does eslint understand with `"agreeto"` that it has to load the package called `eslint-config-agreeto` that's located in `packages/config/eslint`. 

I've tried replacing "agreeto" with the actualy package name and it worked, so have the following suggestion. Could you test that on your end?

```suggestion
  // This tells ESLint to load the config called `eslint-config-agreeto`
 // @see https://turbo.build/repo/docs/getting-started/create-new#understanding-eslint-config-custom
  extends: ["eslint-config-agreeto"], // -> packages/config/eslint
```

Reply:
they do a bunch of this stuff in their config, for example you dont need the `plugin:` prefix in the plugins i believe.

i watched a stream where Joshua Goldberg setup eslint for solid and he mentioned a bunch of "implicit" behavior eslint does when resolving the config